src/naive/time.rs

```diff
@@ -16,6 +16,9 @@ use format::{parse, ParseError, ParseResult, Parsed, StrftimeItems};
 use format::{Fixed, Item, Numeric, Pad};
 use Timelike;
 
+pub const MIN_TIME: NaiveTime = NaiveTime { secs: 0, frac: 0 };
+pub const MAX_TIME: NaiveTime = NaiveTime { secs: 23 * 3600 + 59 * 60 + 59, frac: 1_999_999_999 };
```

Comment:
I think that this frac should be just `999_999_999` since it is a bit unlikely (but not impossible) that the very last second will be a leap second.

Reply:
Ah, I see, thanks.